environment.yml

```diff
@@ -1,17 +1,18 @@
 name: cognoma-machine-learning
 dependencies:
+- conda-forge::altair=1.2.0
+- conda-forge::vega=0.4.4
 - jupyter=1.0.0
-- numexpr=2.6.0
-- numpy=1.11.1
-- pandas=0.18.1
-- python=3.5.2
-- requests=2.11.1
-- scikit-learn=0.18.0
-- scipy=0.17.1
-- seaborn=0.7.0
+- nomkl
```

Comment:
So the error I'm getting on linux without `nomkl`, i.e. with `mkl` is a dead kernel in the notebook and the error:

```
Intel MKL FATAL ERROR: Cannot load libmkl_avx2.so or libmkl_def.so.
```

Will look into alternative solutions. Also cross-referencing https://github.com/ContinuumIO/anaconda-issues/issues/720, https://github.com/BVLC/caffe/issues/3884, https://github.com/scikit-learn/scikit-learn/issues/5046